policy/README.md

```diff
@@ -365,6 +384,130 @@ If a vehicle is matched with a rule, then it _will not_ be considered in the sub
+
+This endpoint it not authenicated (ie. public), and allows the discovery of other public APIs like Geography, Policy, and Jurisdiction. The agency can host this as a file or API on their servers, on a third party server, or the OMF can host on behalf of an agency in the [agency requirements repo](#). See this [hosting guidance document](#) for more information.  This requirements file can be [referenced directly](https://github.com/openmobilityfoundation/governance/blob/main/technical/OMF-MDS-Policy-Language-Guidance.md) in an agency's operating permit/policy document when discussing program data requirements.
+
+See [Policy Requierment Examples](/policy/examples/requirements.md) for how this can be implemented.
+
+An agency's [Requirements](#requirements) endpoint contains a number of distinct parts, namely [metadata](#requirement-metadata) and [MDS version](#requirement-mds-version) (with sub sections on applicable providers and relevant [MDS APIs](#requirement-mds-apis)). 
+
+```jsonc
+{
+  "metadata": {
+    // metadata fields per the "Requirement Metadata" section
+  },
+  "mds_versions" [ 
+    {
+      "version" : "[MDS VERSION NUMBER]",
+      "provider_ids": [
+        // provider id array
+      ],
+      "mds_apis": [
+        {
+          "api_name" : "[MDS API]": {
+            // MDS endpoints, urls, optional fields
+          }
+        },
+        // other MDS APIs per the "Requirement MDS APIs" section
+      ]
+    }, 
+    // other MDS versions per the "Requriement MDS Version" section
+  }
+}
+```
+[Top][toc]
+
+#### Requirement Metadata
+
+Contains metadata applicable to the agency and at the top of its [Requirement](#requirement) data feed in the `metadata` section. 
+
+| Name                         | Type            | Required / Optional | Description              |
+| ---------------------------- | --------------- | -------- | ----------------------------------- |
+| `mds_release`                | text            | Required | Release of MDS that the requirements data feed aligns to, based on official MDS releases. E.g. "1.2.0" |
+| `version`                    | integer         | Required | Version of this file. Increment 1 with each modification. E.g. "3" |
+| `last_updated`               | [timestamp][ts] | Required | When this file `version` was last updated. E.g. "1611958740" |
+| `max_update_frequency`       | integer         | Required | The expected maximum frequency with which this file could be updated. E.g. "P1D" |
+| `omf_review`                 | Enum            | Required | yes/no. Was this file reviewed by OMF Staff for accuracy? E.g. "yes" |
```

Comment:
Sorry if I'm misunderstanding the original impetus for this, and I know that this field is being removed for now, but would have some kind of official OMF-sanctioned validation tool for the requirements endpoint payload suffice instead?

Reply:
Yes that would also work, but building and maintaining a tool would be a much bigger undertaking, and may still have the same implications here, eg. "this feed was validated by the OMF tool", which could still be a liability issue. Not sure so removing for now. Instead it could just be a "validator" text field that says who or what validated the feed, like 'Ride Report' or 'Populus' or 'OMF Validator', etc.

Reply:
So we can add a 'feed_validator' text field which would just be the free form name of the organization that validated the tool. No need to enumerate values.  And a second 'feed_validator_date' field that is the ISO date (eg 2021-06-02) of when the most recent validation was done, useful to compare the last updated date of the feed to this date.

Reply:
It think we are going to leave this off for the initial release. I'd like to wait and see what sort of adoption we get before building validating tools.